TJDropbox/TJDropbox.m

```diff
@@ -354,6 +339,33 @@ + (NSString *)asciiEncodeString:(NSString *const)string
     return result;
 }
 
+#pragma mark - Token Migration
+
++ (void)migrateV1TokenToV2Token:(NSString *const)accessToken accessTokenSecret:(NSString *const)accessTokenSecret appKey:(NSString *const)appKey appSecret:(NSString *const)appSecret completion:(void (^const)(NSString *_Nullable, NSError *_Nullable))completion
+{
+    // https://github.com/dropbox/dropbox-sdk-obj-c/blob/3769fbacb0b458de7c20db08edba6ca21c54b650/Source/ObjectiveDropboxOfficial/Shared/Handwritten/OAuth/DBSDKKeychain.m#L363-L453
+    // https://www.dropbox.com/developers/documentation/http/documentation#auth-token-from_oauth1
+    
+    NSMutableURLRequest *const request = [NSMutableURLRequest requestWithURL:[NSURL URLWithString:@"https://api.dropboxapi.com/2/auth/token/from_oauth1"]];
```

Comment:
You could use `+apiRequestWithPath:accessToken:parameters:` to create the request complete with the host, `Content-Type`, and body you provided here. You'll need to modify `accessToken` to be optional, but otherwise would work.